pyEpiabm/pyEpiabm/property/spatial_foi.py

```diff
@@ -96,11 +98,25 @@ def spatial_susc(susc_cell, infector, infectee, time: float):
             spatial_susc = pyEpiabm.core.Parameters.instance().\
                 age_contact[infectee.age_group]
 
-        if infector.microcell.closure_start_time is not None:
-            spatial_susc *= Parameters.instance().\
-                intervention_params['place_'
-                                    'closure']['closure_spatial_params']
-        return spatial_susc
+        closure_susc = Parameters.instance().\
+            intervention_params['place_closure']['closure_spatial_params'] \
+            if infector.close_place(
+                Parameters.instance().intervention_params['place_closure'][
+                    'closure_place_type']) is True else 1
+
+        if (hasattr(infector.microcell, 'distancing_start_time') is True) and (
+                infector.microcell.distancing_start_time is not None):
+            if infector.distancing_enhanced is True:
+                distancing_susc = Parameters.instance().\
+                    intervention_params['social_distancing'][
+                        'distancing_spatial_enhanced_susc']
+            else:
+                distancing_susc = Parameters.instance().\
+                    intervention_params['social_distancing'][
+                        'distancing_spatial_susc']
+        else:
+            distancing_susc = 1
```

Comment:
Would it be more clear to have distancing_susc = 1 above and than multiply if needed in the if loop. So similar as for spatial_susc?

Reply:
Good point! I also modified in household_foi.py.